src/AutoBuild.jl

```diff
@@ -1379,7 +1379,7 @@ function build_project_dict(name, version, dependencies::Array{Dependency}, juli
         "deps" => Dict{String,Any}(),
         # We require at least Julia 1.3+, for Pkg.Artifacts support, but we only claim
         # Julia 1.0+ by default so that empty JLLs can be installed on older versions.
-        "compat" => Dict{String,Any}("JLLWrappers" => "1.2.0",
+        "compat" => Dict{String,Any}("JLLWrappers" => "1.3.0",
```

Comment:
Technically, I don't think we even need to bump the required version, as the API will be the same, which makes this pull request useless